.github/workflows/check-for-update.yaml

```diff
@@ -1,101 +1,53 @@
-name: check-for-update
+name: update-prisma-version
```

Comment:
Please create a new, additional workflow without touching the existing implementation to much for now.